docs/books/admin_guide/13-softwares.md

```diff
@@ -900,3 +900,121 @@ Is this ok [y/N]:
 ### Conclusion
 
 EPEL is not an official repository for RHEL. But it can be useful for administrators and developers who work with RHEL or derivatives and need some utilities prepared for RHEL from a source they can feel confident about.
+
+## DNF Plugins
+
+The `dnf-plugins-core` package adds plugins to `dnf` that will be useful for managing your repositories.
+
+!!! NOTE
+    See more informations here: https://dnf-plugins-core.readthedocs.io/en/latest/index.html
+
```

Comment:
Hey @alemorvan, for the most part this looks good. I didn't have a chance to test anything and no time to really look hard at it. I *think* that you and I decided that for admonitions we needed the content to be one line down from the admonition type like this:
```
!!! Note
    
    See more information here ...
```
If I recall, this had something to do with how Crowdin messed with admonitions.

Reply:
I did the change.
Thx